flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/functions/sql/FlinkSqlOperatorTable.java

```diff
@@ -776,7 +776,8 @@ public SqlMonotonicity getMonotonicity(SqlOperatorBinding call) {
 	 * We need custom group auxiliary functions in order to support nested windows.
 	 */
 	public static final SqlGroupedWindowFunction TUMBLE = new SqlGroupedWindowFunction(
-			SqlKind.TUMBLE, null,
+			// The TUMBLE group function was hard code to $TUMBLE in CALCITE-3382.
+			"$TUMBLE", SqlKind.TUMBLE, null,
```

Comment:
Only TUMBLE, what about the other functions?

Reply:
Only the `TUMBLE` function changed, because it is generated from the parser directly. We can not really control its name.